src/openrct2/drawing/drawing.h

```diff
@@ -363,6 +363,8 @@ void shorten_path(utf8 *buffer, size_t bufferSize, const utf8 *path, sint32 avai
 void scrolling_text_initialise_bitmaps();
 sint32 scrolling_text_setup(rct_string_id stringId, uint16 scroll, uint16 scrollingMode);
 
+void FASTCALL gfx_get_sprite_size(sint32 image_id, sint32* width, sint32* height);
```

Comment:
@IntelOrca 
I see a `#pragma pack(push)` and `#pragma pack(pop)` in drawing.h, and the majority of typedefs inside drawing.h are confined inside the packings, the only exception being
```
typedef struct rct_palette {
    rct_palette_entry entries[256];
} rct_palette;
```
The rct_size16 will go inside the packing as well, right?